### DISCUSSION TITLE
Zoom

### DISCUSSION THREAD

**Discussion by mrmaly** [nurkiewicz/polski-w-it#244] [Ideas]
Technically, "zoom" to działa w obie strony, nie można chyba mieć hasła "zoom" - "powiększać, powiększenie".
"zoom in" to jest powiększać, "zoom out" to pomniejszać.
Rzeczowniki to to chyba nie są, bo trzeba by było albo zapisać je z hyphenem, albo jego gerunds.
Czyli się robią cztery hasła z jednego...

**Comment by Tomirad** [comment]
zoom to poziom powiększenia
zoom x1 - to powiększenie normalne/normatywne
zoom x2 - to powiększenie podwójne
zoom x1/2 to - to powiększenie dwa razy mniejsze, pomniejszenie
